fw/http.h

```diff
@@ -470,6 +473,7 @@ struct tfw_http_req_t {
 	TfwHttpCond		cond;
 	TfwMsgParseIter		pit;
 	TfwStr			userinfo;
+	TfwStr			*uri_host;
```

Comment:
Don't we have an alignment hole due to this member placement?

Reply:
Nope. TfwStr starts itself with a pointer, so no alignment gaps here.

Reply:
Nope. TfwStr has 8 byte alignment